.travis.yml

```diff
@@ -52,8 +50,8 @@ addons:
       
 before_install:
   - go get -u github.com/golang/lint/golint honnef.co/go/tools/cmd/...   
-  - if (("$TRAVIS_OS_NAME" == "osx") && (!"$SKIP_GRAPHVIZ")); then brew update          ; fi
-  - if (("$TRAVIS_OS_NAME" == "osx") && (!"$SKIP_GRAPHVIZ")); then brew install graphviz; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" && (-z $SKIP_GRAPHVIZ) ]]; then brew update          ; fi
```

Comment:
I think ( and ) are not needed.

Reply:
This seems to work.
https://travis-ci.org/nolanmar511/pprof/builds/339596412
Build 301.8 has output after `if [[ "$TRAVIS_OS_NAME" == "osx" && -z $SKIP_GRAPHVIZ ]]; then brew install graphviz; fi` that's

Reply:
Done.